Added more ship names for Amazon tribes

alasiax opened the pull request:
Based on rivers in the Amazon Basin for naming consistency.

alasiax committed:
Added more ship names for Amazon tribes

Based on rivers in the Amazon Basin for naming consistency.

frankystone reviewed (changes requested):
Thanks for contributing!

There are some double entries, please fix.

alasiax committed:
Updated 

Done without duplicates this time. Removed Roosevelt as it doesn't fit the theme, and a couple of others which are close matches eg Arauá/Arauã, Guaribe/Guariba

alasiax commented:
Thanks Frankystone :o) Updates made as requested.

alasiax committed:
Merge branch 'master' into patch-1

frankystone reviewed (approved):
LGTM :-)

Let's wait for the checks to finish.

hessenfarmer commented:
Nice work thanks a lot
